applications/call_inspector/src/call_inspector_maintenance.erl

```diff
@@ -80,16 +80,16 @@ callid_details(CallId) ->
                                     ,{call_inspector, fun kapi_inspector:lookup_resp_v/1, true}
                                     )
     of
-        {'ok', JObjs} -> print_jobjs(JObjs);
-        {'timeout', []} ->
+        {ok, JObjs} -> print_jobjs(JObjs);
```

Comment:
Please revert. Atoms should still be tickied despite the lackadaisical attitude and changing this for no reason isn't appropriate anyway.